From 853cdd6d8667a19abf79444cda1d4eaa0102b068 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 19 Nov 2020 09:54:20 +0100
Subject: [PATCH 54/73] build: fix silly preprocessor error line

ui/aura/window.h:45:67: warning: missing terminating ' character
   45 | #error This file must not be included on macOS; Chromium Mac doesn't use Aura.
---
 ui/aura/window.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git ui/aura/window.h ui/aura/window.h
index d303446985150..3eb512cd411ed 100644
--- ui/aura/window.h
+++ ui/aura/window.h
@@ -43,7 +43,7 @@
 #include "ui/gfx/native_widget_types.h"
 
 #if defined(OS_APPLE)
-#error This file must not be included on macOS; Chromium Mac doesn't use Aura.
+#error This file must not be included on macOS; Chromium Mac does not use Aura.
 #endif
 
 namespace cc {
-- 
2.32.0

